rpc_jobs/standard_job_dep_update.yml

```diff
@@ -0,0 +1,66 @@
+- job-template:
+    CRON: "@weekly"
+    RPC_GATING_BRANCH: master
+    NUM_TO_KEEP: 30
+    READ_ONLY_TEST: true
+    SLAVE_TYPE: "container"
+    name: '{trigger}-Dep-Update_{repo_name}-{branch}'
+    project-type: pipeline
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/rcbops/rpc-gating
+            branches:
+              - "${{RPC_GATING_BRANCH}}"
+            credentials-id: "github_account_rpc_jenkins_svc"
+      script-path: job_dsl/standard_job_dep_update.groovy
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: "{NUM_TO_KEEP}"
+      - github:
+          url: "{repo_url}"
+      - inject:
+          properties-content: |
```

Comment:
Nice!  I think I prefer this to the arbitrary env var assignment we typically do in the dsl.

Reply:
Yes, I thought this looked a bit nicer too. Maybe worth considering applying to the other jobs at some point.